p8e-api/src/main/kotlin/io/provenance/engine/service/ChaincodeInvokeService.kt

```diff
@@ -49,6 +49,14 @@ class ChaincodeInvokeService(
     private val chaincodeProperties: ChaincodeProperties,
     private val provenanceGrpc: ProvenanceGrpcService,
 ) : IChaincodeInvokeService {
+
+    companion object {
+        private val blockScopeIds = HashSet<String>()
+        fun freeScope(scopeUuid: UUID) {
```

Comment:
free in this context the same as release/remove scopes?

Reply:
yeah, maybe release would be a better word, or unlock 🤔

Reply:
changed everything to lock/unlock